src/ray/gcs/gcs_server/actor_info_handler_impl.cc

```diff
@@ -0,0 +1,84 @@
+#include "actor_info_handler_impl.h"
+#include "ray/util/logging.h"
+
+namespace ray {
+namespace rpc {
+
+void DefaultActorInfoHandler::HandleGetActorInfo(
+    const rpc::GetActorInfoRequest &request, rpc::GetActorInfoReply *reply,
+    rpc::SendReplyCallback send_reply_callback) {
+  RAY_LOG(DEBUG) << "Begin get actor info, actor id is:" << request.actor_id();
+  ActorID actor_id = ActorID::FromBinary(request.actor_id());
+  auto on_done = [request, reply, send_reply_callback](
+                     Status status, const boost::optional<ActorTableData> &result) {
+    if (status.ok()) {
+      if (result) {
```

Comment:
we can change this `if` to an `assert`, because if status is ok, result should not be none.

Reply:
fixed